TELEMETRY.md

```diff
@@ -2,9498 +2,3 @@
 
 Expand each section to see more information about that event.
 
-<details>
```

Comment:
Yeah looks like telemetry generation is broken. You likely need to change the telemetry generator too.

Reply:
It seems the generator fails to work after moving files around, fix it now.